ras_party/controllers/controller.py

```diff
@@ -33,6 +34,10 @@
     REQUESTS_GET_TIMEOUT = 20
```

Comment:
I'm not sure this code is part of the PR, but I've never seen it before, and am wondering why we need to catch a generic Exception here? AFAICT, there's only a ValueError to worry about, which we can just allow to propagate. Also, we really need to avoid having direct access into the environment / file system from arbitrary locations in the code. There's an existing mechanism to inject config from the top of the process.